src/constants.js

```diff
@@ -0,0 +1,5 @@
+export const API_KEY = '8cac6dec66e09ab439c081b251304443'
+export const ENDPOINT = 'https://api.themoviedb.org/3'
```

Comment:
there are likely to be different environments out there for testing and production versions. Please use .env file to configure links.
API key is something unique to us and should not be exposed to the frontend at all